src/main.py

```diff
@@ -1,11 +1,102 @@
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+import pymongo
+
+from outlierhandler import OutlierHandler
 import file_reader_h5
 import mongoConnection
 
 
+def handle_outliers_and_prepare_for_db_push():
```

Comment:
@felixkru pack sowas bitte nicht in die main.py, das kannst du besser woanders machen, z.B. im outlinerhandler oder ggf. in einem Programmcontroller